### ISSUE TITLE
Bad performance when browsing a WMTS layer with uncomplete tilegrid

### ISSUE THREAD

**Issue by thhomas** [openlayers/ol3#4383]
Here is an [example](http://jsfiddle.net/thhomas/Lrew2r7d/) where I try to display a WMTS layer with a tilegrid restricted to matrixIds and resolutions starting at 6 instead of 0. 
When I zoom out beyond the level 6, ol keeps loading the last level to fill the view and it could mean a lot of tiles when I keep zooming out. It could crash the web browser.
Maybe some limitations should be set to prevent ol from loading too much tiles and crashing the browser.

**Comment by ahocevar** [comment]
You can set such limits yourself in the ol.View configuration, by using something like `minZoom: 6`.

**Comment by thhomas** [comment]
Ok, changing view configuration is a way to solve this issue.
It would mean, though, that I have control the minZoom view when I switch between layers with different tilegrid.

**Comment by tsauerwein** [comment]
Can't you set `minResolution` and `maxResolution` on the layer?
http://openlayers.org/en/master/apidoc/ol.layer.Layer.html

**Comment by thhomas** [comment]
Thanks @tsauerwein, I prefer this solution.

**Comment by ahocevar** [comment]
If you want to use tiles from zoom level 6 to display them at zoom level 0, there is no other way than loading all these tiles, if you want to display the layer.

**Comment by thhomas** [comment]
My layer has a tileGrid with levels form 18 to 6, I want ol3 to display the layer in levels 18 to 6 (included) and not displaying it at all at levels 5 to 0. It is all connected to the pull request #4374 I made to use extra information in getcapabilities. I managed to get the expected behaviour using maxResolution setting set to `options.tileGrid.getResolution(options.tileGrid.getMinZoom()) * 2`. I updated the [example](http://jsfiddle.net/thhomas/Lrew2r7d/) to illustrate this.
